WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationLocalStore.swift

```diff
@@ -239,6 +273,35 @@ public final class ConversationLocalStore: ConversationLocalStoreProtocol {
         }
     }
 
+    public func addSystemMessage(
+        _ message: SystemMessage,
+        to conversation: ZMConversation
+    ) async {
```

Comment:
This was factored out of the ZMSystemMessage legacy extension, so we can use it directly from the repo. There are other places in WireDomain where we still use the legacy helper method that will be refactored as well if this change is approved.

Reply:
Note: This will eventually be removed from here when this [PR](https://github.com/wireapp/wire-ios/pull/2096) is merged.